frontend/src/components/worksheets/NewRun/NewRun.jsx

```diff
@@ -165,7 +165,7 @@ class NewRun extends React.Component<{
         name: 'untitled-run',
         description: '',
         tags: [],
-        disk: kDefaultDisk,
+        disk: '',

```

Comment:
If it just needs a frontend change it's good to go

Nit: Remove the kdefault constant declaration on line before merging, since it's not used anymore

Reply:
Thanks Yipeng!